package.json

```
@@ -2,5 +2,10 @@
   "name": "linea-ens",
   "version": "1.0.0",
   "private": true,
-  "description": "Linea ENS monorepo"
+  "description": "Linea ENS monorepo",
+  "packageManager": "pnpm@9.15.4+sha512.b2dc20e2fc72b3e18848459b37359a32064663e5627a51e4c74b2c29dd8e8e0491483c3abb40789cfd578bf362fb6ba8261b05f0387d76792ed6e23ea3b1b6a0",
+  "devDependencies": {
+    "@lavamoat/allow-scripts": "^3.4.0",
+    "@lavamoat/preinstall-always-fail": "^2.1.1"
+  }
```

```suggestion
  },
  "scripts":{
    "lifecycle:run": "pnpm -r exec allow-scripts",
    "lifecycle:scan": "pnpm -r exec allow-scripts auto"
  }
```

To manage dependencies in the monorepo, we need to rely on pnpm to run allow-scripts in workspace folders.

lifecycle:run needs to be executed after clean installation of dependencies
lifecycle:scan will add new entries to the allowlist (false by default) after you've installed new packages